app/frontend/.env.production

```diff
@@ -1,3 +1,4 @@
+IS_BETA_ENABLED=false
```

Comment:
I think all environment variables are strings, so even if you set it as `false`, it will still be truthy. I think the solution is to not set it/have this line in this file for production.